.github/workflows/cncf-conformance-test.yaml

```diff
@@ -0,0 +1,58 @@
+name: CNCF Conformance Nightly Latest/Edge Tests
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # Runs every monday midnight
+  pull_request:
+permissions:
+  contents: read
+
+jobs:
+  test-cncf-conformance:
+    name: CNCF Conformance ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.releases }}
+    strategy:
+      matrix:
+#        os: ["ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04"]
+        os: ["ubuntu:24.04"]
+        arch: ["amd64"]
+        releases: ["latest/edge"]
```

Comment:
The classic snap is not CNCF conformant right now because of Cilium.
Please only test moonray (`latest/edge/moonray`) for now and add a TODO to enable this test again once we upgraded Cilium to 1.17